include/swift/SILOptimizer/Analysis/ArraySemantic.h

```diff
@@ -131,6 +134,10 @@ class ArraySemanticsCall {
   /// Returns true on success, false otherwise.
   bool replaceByValue(SILValue V);
 
+  /// Replace a call to append(contentsOf: ) with a series of
+  /// append(element: ) calls.
+  bool replaceByAppendingValues(SILModule &M, SILFunction* appendFn, llvm::SmallVectorImpl<SILValue> &Vs, ArrayRef<ProtocolConformanceRef> &conformances);
```

Comment:
Stylistic nitpicks: lines should be <= 80 columns, and put the * and & before the type and not after the name.